rebar.config

```diff
@@ -31,24 +31,24 @@
 %% git config --global url."git://github".insteadOf https://github
 {deps,
  [
-  {base16, ".*", {git, "https://github.com/goj/base16.git", "f78918e"}},
+  {base16, ".*", {git, "https://github.com/goj/base16.git", {ref, "f78918e"}}},
   {exml, ".*", {git, "https://github.com/esl/exml.git", {tag, "3.0.1"}}},
-  {lager, ".*", {git, "https://github.com/erlang-lager/lager.git", {ref, "06f676a"}}},
-  {lager_syslog, ".*", {git, "https://github.com/erlang-lager/lager_syslog.git", "3.0.3"}},
-  {cowboy, ".*", {git, "https://github.com/ninenines/cowboy.git", "1.1.2"}},
+  {lager, "3.6.4"},
```

Comment:
If we're moving to hex here, why not also `{base16, "1.0.0"}`, `{cowboy, "1.1.2"}`, `{idna, "5.1.2"}`, `{uuid, "1.7.3", {pkg, uuid_erl}}`, `{riakc, "2.5.3"}`? We could also probably bump `lasse` to `{lasse, "1.2.0"}` and more.

Reply:
* I moved to hex for these packages I actually updated in the PR. Moving to hex with other deps is possible, probably in another PR.
* Regarding `uuid`, I wanted to use the version from `hex` only I had issues with other dep which uses `uuid` from git. I didn't try `{uuid, "1.7.3", {pkg, uuid_erl}}`.
* I'll switch `idna` to use hex
* We are not able to use `lasse` `1.2.0` yet, because it depends on `cowboy` 2.x
* Regarding other deps, I tried updating them as well. They introduced some random failures in our test and I didn't have time to investigate these failures yet. What we have in this PR is the minimal stable upgrade of deps I was able to do in one day.

Reply:
The packages I mentioned are all upgraded in the PR. Sorry about the confusion with `lasse` version, please see the updated comment. :)

Reply:
And yeah, plain `uuid` package in Hex is an Elixir library; `uuid_erl` is the one we're using in Mongoose.